README.md

```diff
@@ -1 +1,55 @@
-# API-based-web-app
\ No newline at end of file
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet) ![HTML](https://img.shields.io/badge/-HTML-orange) ![CSS](https://img.shields.io/badge/-CSS-blue) ![JavaScript](https://img.shields.io/badge/-JavaScript-yellow)
+
+# JavaScript group capstone: Meal Mentor
+
+![Project Screenshot](./snapshot.PNG "Meal Mentor")
```

Comment:
- [ ] [REQUIRED] As stated in the project requirement, kindly ensure that you make a video presenting the features of the project maximum length of 5 minutes. All the members in the video should appear in the video and talk about the project and the video should contain a highlight about a piece of the code the you are proud of making.

Reply:
Done